coap-gateway/service/observation/observation.go

```diff
@@ -6,15 +6,14 @@ import (
 
 	"github.com/plgd-dev/device/schema"
 	"github.com/plgd-dev/device/schema/interfaces"
-	"github.com/plgd-dev/device/schema/resources"
 	coapMessage "github.com/plgd-dev/go-coap/v2/message"
 	"github.com/plgd-dev/go-coap/v2/message/codes"
 	"github.com/plgd-dev/kit/v2/codec/cbor"
 )
 
 // Query /oic/res resource to get device resource links.
```

Comment:
Docu no longer correct, other resources that support `if.ll` interface can now be queried too.

Reply:
fixed